static/scss/program-page.scss

```diff
@@ -220,6 +226,19 @@
       padding: 40px 5% 140px;
     }
 
+    .faculty-copy {
+      width: 100%;
+      height: 160px;
+      position: absolute;
+      bottom: -1px;
+      padding: 25px 22px;
+      z-index: 2;
+      background: -moz-linear-gradient(top,  rgba(0,0,0,0) 0%, rgba(0,0,0,1) 100%); /* FF3.6-15 */
+      background: -webkit-linear-gradient(top,  rgba(0,0,0,0) 0%,rgba(0,0,0,1) 100%); /* Chrome10-25,Safari5.1-6 */
+      background: linear-gradient(to bottom,  rgba(0,0,0,0) 0%,rgba(0,0,0,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
```

Comment:
there's a couple things that can automatically add all of these vendor prefixes, would it be helpful for us to set those up?